tasks/utils.ts

```diff
@@ -1,5 +1,9 @@
 import { subtask } from 'hardhat/config';
-import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { HardhatRuntimeEnvironment, Libraries } from 'hardhat/types';
+import * as path from 'path';
+import * as prettier from 'prettier';
+import * as fs from 'fs';
+import { tscompile } from '../utils/tscompile';
 
```

Comment:
@henryacaron why are we adding all of these deploy tasks to utils? Can we put them in a new file for cleanliness sake?

Reply:
they are deploy utilities so i figured they belonged best in utils